pyproject.toml

```diff
@@ -15,9 +15,9 @@ classifiers = [
 python = ">=3.9.0,<3.11"
 numpy = "^1.24"
 pandas = "^2.0"
-flwr = "1.4.0"
+flwr = "1.7.0"
 opacus = "^1.3.0"
-torch = "^1.12.1"
+torch = "2.0.0"
```

Comment:
Did we want 2.1.1 here rather than 2.0.0?

Reply:
I run into some reproducibility issues when I upgrade to torch 2.1.1 that I ran into. Instead of pinning to ^2.1.1, do you think its okay to use 2.0.0. Thats what I have been able to get working in terms of consistency between my local machine and the github server

Reply:
Hashing out with Marcelo now